patches/api/0270-EntityMoveEvent.patch

```diff
@@ -18,10 +18,13 @@ index 0000000000000000000000000000000000000000..fae0b1daf733e85d8f691f5ef934b8c0
 +import org.bukkit.event.Cancellable;
 +import org.bukkit.event.HandlerList;
 +import org.bukkit.event.entity.EntityEvent;
++import org.bukkit.event.player.PlayerMoveEvent;
```

Comment:
you can use fully qualified imports for javadocs (though if you choose not to then this should have a `// Paper`)

Reply:
This is a paper event class though, so no // Paper is needed

Reply:
OH lol sorry 😅